docs/docs/dev_guide/docs.md

```diff
@@ -1,12 +1,38 @@
 ---
 layout: default
 title: Developing Docs
-parent: dev_guide
+parent: Developer's Guide to AaC
 nav_order: 2
 ---
-### Documenting
 
-#### Writing API Documentation
+# AaC Documentation
+AaC documentation consists of two groups of generated documents: these jekyll pages and Sphinx API documentation for the Python API.
+
+## Jekyll Docs
+Jekyll docs are manually-maintained readme pages. We use these Jekyll docs to document topics such as user-facing documentation and developer-facing documentation (like this page).
+
+### Setup
+The Jekyll docs use the Ruby build tool [Bundler](https://bundler.io/). You'll need to have this installed in your local development environment.
+
+If you haven't built the Jekyll docs before, you'll need to run `bundle install` inside the top-level repository directory `docs`. Voila! The links work, and you can create a public link to share with others to see.
+### Starting the Preview Server
+
+
+### Serving Jekyll Pages in Gitpod
+Because Gitpod is a container-based development environment, it present some peculiarities for users who may be trying to server Jekyll files. One of the larger issues is that the Jekyll docs are using localhost or `0.0.0.0` as the base URL while the actual hostname is the Gitpod container's public address.
```

Comment:
```suggestion
Because Gitpod is a container-based development environment, it presents some peculiarities for users who may be trying to serve Jekyll files. One of the larger issues is that the Jekyll docs use `localhost` or `0.0.0.0` as the base URL while the actual hostname is the Gitpod container's public address.
```